tiles/src/main/java/com/protomaps/basemap/layers/Buildings.java

```diff
@@ -34,12 +58,21 @@ public void processFeature(SourceFeature sf, FeatureCollector features) {
 
       var feature = features.polygon(this.name())
         .setId(FeatureId.create(sf))
-        .setAttrWithMinzoom("building:part", sf.getString("building:part"), 13)
+        .setAttr("pmap:kind", kind)
         .setAttrWithMinzoom("layer", sf.getString("layer"), 13)
-        .setAttrWithMinzoom("height", height, 13)
-        .setZoomRange(10, 15);
+        // NOTE: Height is quantized by zoom in a post-process step
+        .setAttr("height", height)
+        .setZoomRange(min_zoom, 15);
+
+      if( kind == "building_part") {
+        // We don't need to set WithMinzoom because that's implicate with the ZoomRange
+        feature.setAttr("pmap:kind_detail", sf.getString("building:part"));
+        feature.setAttr("min_height", sf.getString("min_height"));
+      }
 
-      OsmNames.setOsmNames(feature, sf, 13);
+      // Names should mostly just be for POIs
+      // Sometimes building name and address are useful items, but only at zoom 17+
+      //OsmNames.setOsmNames(feature, sf, 13);
```

Comment:
Drop the names from buildings... if it's really important it'll show up in the POIs layer.

Alternatively we could only set this starting at zoom 15? As some building names will be lost (when there isn't a business there).